third_party/pyth/p2w_autoattest.py

```diff
@@ -148,9 +156,27 @@ def serve_attestations():
         logging.error("Bad Content type")
         sys.exit(1)
 
-    PYTH_PRICE_ACCOUNT = pyth_accounts["price"]
-    PYTH_PRODUCT_ACCOUNT = pyth_accounts["product"]
-    logging.info(f"Retrieved Pyth accounts from endpoint: {pyth_accounts}")
+    cfg_yaml = f"""
+---
+symbols:"""
```

Comment:
FYI python has a `yaml` library that will generate the yaml text from python arrays / dictionaries. I think that's better than generating the string yourself here. docs: https://pyyaml.org/wiki/PyYAMLDocumentation

Reply:
I did not consider it worth it. This YAML's schema is insignificant and I expect that to be the case for forseeable future.